### PR TITLE
workaround for localizing timestamps with timezone abbreviations

### PR DESCRIPTION
**Pull request by karlicoss**

### CODE REVIEW

File: google_takeout_parser/parse_html/html_time_utils.py [hunk 1]
```diff
@@ -41,28 +41,64 @@ def abbr_to_timezone(abbr: str) -> tzinfo:
 _TIME_FORMAT = "%b %d, %Y, %I:%M:%S %p"
 
 
-def parse_html_dt(s: str) -> datetime:
```
Comment (karlicoss) [inline google_takeout_parser/parse_html/html_time_utils.py:44]: technically, this is an API change, so could default it to `None`, but don't think anything else is using it?

File: google_takeout_parser/parse_html/activity.py [hunk 2]
```diff
@@ -53,6 +53,8 @@ def _group_by_brs(els: TextOrEl) -> ListOfTags:
 
 def _parse_subtitles(
     subtitle_cell: bs4.element.Tag,
+    *,
+    file_dt: Optional[datetime],
```
Comment (karlicoss) [inline google_takeout_parser/parse_html/activity.py:57]: could default to `None` here, but they are not exposed to the public/only used in tests, so figured safer to be verbose

Comment (seanbreckenridge) [reply google_takeout_parser/parse_html/activity.py:57]: yeah, sounds good

File: google_takeout_parser/parse_html/html_time_utils.py (hunk 1 above)
Comment (seanbreckenridge) [reply google_takeout_parser/parse_html/html_time_utils.py:44]: yeah, I think its fine. Fine for there to be breaking changes for this project at this point, nothing really set in stone yet

**Review by seanbreckenridge — APPROVED** [review]
looks great, thanks!

I know this was a pain to debug, really appreciate it

**Pull request merged**